### ISSUE TITLE
Staging Review finding: Add heading to the card

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#116267]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Management Tool 2
**Product:** Claim Status Tool
**Feature:** VA Letters and Documents

## Findings details
**VA.gov Experience Standard - issue:** [User encounters consistent, clear design components and patterns.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @allison0034 (Design), @erinrwhite (IA), @rsmithadhoc (Accessibility)

### Description

There is a card on the page with the mailing address and edit button but no heading inside the card. Cards should have headings.

<details><summary>Link, screenshot or steps to recreate</summary>

![profile](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/98499-allison-1.png?raw=true)

</details>

## Recommended action

Look at your content and the flow of things, you could possibly just move the “Mailing address” into the card if that makes sense. This will also move up the actionable item. If that solution will not work, work with content and IA to figure out how to add a heading without having redundant text.

IA Feedback: I encourage using active language for the current "Mailing address" heading - something like "Verify your mailing address" - and then, within the card, add a subheading that says "Your mailing address".

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** [WCAG 2.4.6 Headings and Labels - Level AA](https://www.w3.org/TR/WCAG22/#headings-and-labels)
- **Modality:** N/A
- **Design System Component:** [Card component](https://design.va.gov/components/card)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** Page headings/heading hierarchy

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
